FantasmoSDKExample/ViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ViewController.swift
+//  FantasmoSDKExample
+//
+//  Created by Nick Jensen on 29.09.21.
+//
```

Comment:
This entire example app could be removed. This`ViewController` code could just be moved to the README.